Gruntfile.js

```diff
@@ -21,7 +21,7 @@ module.exports = function(grunt) {
         },
         watch: {
             scripts: {
-                files: ['js/app.js', 'app/views/*'],
+                files: ['js/app.js', 'app/views/**'],
```

Comment:
Without this change, Grunt doesn't re-render the templates in `app/views/main` in the watch mode.